lib/IDE/Refactoring.cpp

```diff
@@ -2752,78 +2752,107 @@ bool RefactoringActionLocalizeString::performChange() {
   return false;
 }
 
+struct MemberwiseParameter {
```

Comment:
Comments would be nice, if you could specify the invariants (DefaultExpr night be null, Type must be valid, Name must be valid)